iset.mm

```diff
@@ -130285,6 +130293,255 @@ we show (in ~ tgcl ) that ` ( topGen `` B ) ` is indeed a topology (on
     ( ctop wcel c0 cnei cfv 0opn opnneiid mpbird ) ABCDDAEFFCDACAGADHI $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Subspace topologies
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d j x y $.
+    $( Reverse closure for the subspace topology.  (Contributed by Mario
+       Carneiro, 19-Mar-2015.)  (Proof shortened by Jim Kingdon,
+       23-Mar-2023.) $)
+    restrcl $p |- ( ( J |`t A ) e. Top -> ( J e. _V /\ A e. _V ) ) $=
+      ( vj vx vy crest co ctop wcel c0 cvv wa 0opn cv cin cmpt df-rest elmpt2cl
+      crn syl ) BAFGZHIJUAIBKIAKILUAMCDKKECNENDNOPSBAFJDECQRT $.
+  $}
+
+  ${
+    $d a b c u v w x y z A $.  $d a b c u v w x y z B $.  $d x J $.  $d x X $.
+    $d w x y z V $.  $d w x y z W $.
+    $( A subspace topology basis is a basis. ` Y ` is normally a subset of the
+       base set of ` J ` .  (Contributed by Mario Carneiro, 19-Mar-2015.) $)
```

Comment:
What is meant by `J`and `Y` here? These class variables do not appear in the theorem.

Reply:
Fixed in both set.mm and iset.mm.